apps/meteor/tests/end-to-end/api/livechat/10-departments.ts

```diff
@@ -254,25 +260,30 @@ describe('LIVECHAT - Departments', function () {
 							expect(res.body.items).to.have.length.of.at.least(1);
 							expect(res.body.items[0]).to.have.property('_id');
 							expect(res.body.items[0]).to.have.property('name');
+							deleteDepartment(res.body.items[0]._id);
 						})
 						.end(done);
 				});
 		});
-
-		it('should return a list of departments excluding the ids on selector.exceptions', (done) => {
-			let dep: ILivechatDepartment;
+		(IS_EE ? it : it.skip)('should return a list of departments excluding the ids on selector.exceptions', (done) => {
```

Comment:
this looks much cleaner IMO
```suggestion
		it('should return a list of departments excluding the ids on selector.exceptions', (done) => {
			if (!IS_EE) {
				this.skip();
			}
```

Reply:
We should have an `it.ee` helper that does this check auto imho.

But for now, this works 👀

Reply:
`this.skip()` does not work on arrow functions, should I convert the arrow function to a normal function in order to do the `this.skip()`?

![Captura de Tela 2023-02-03 às 10 57 00](https://user-images.githubusercontent.com/20868078/216621641-ea5d82ee-fea2-45d0-8e20-5036599b2010.png)

Reply:
yes pls, change it to regular function.. IMO arrow functions should **only** be used when you need to use `this` from above context. 😬